docs/evm/fees.md

```diff
@@ -9,13 +9,85 @@ sidebar_position: 4
 Are you a Cadence developer looking for information about Fees on Cadence? If so, check out the Cadence specific documentation [here](../build/basics/fees.md)
 
 :::
-
-# Fees
+<h2>Fees</h2>
 
 EVM transactions are ultra low-cost and use the native FLOW token as gas. [Externally Owned Accounts (EOAs)](https://developers.flow.com/evm/build/accounts) function the same on Flow as other EVM networks like Ethereum.
 
-**Gasless Transactions**
+<details>
```

Comment:
I think we are overdoing it with the expandable sections :)
If you look at the preview - https://docs-psucmflu5-onflow.vercel.app/evm/fees, the whole page is nothing but expandable sections (very different from the other pages under EVM).

Lets have this section as a regular section.